04.class/lib/view.js

```diff
@@ -0,0 +1,29 @@
+import enquirer from "enquirer";
+
+export class View {
+  #MODE_TABLE = { r: "see", d: "delete" };
+
+  printMemos(memos) {
+    memos.forEach((memo) => {
+      console.log(memo.title);
+    });
+  }
+
+  displayMemoSelectionPrompt(memos, mode) {
+    const titles = memos.map((memo) => memo.title);
```

Comment:
```suggestion
    const titles = memos.map(memo => memo.title);
```

好みの問題ですが引数が一つの時は（）を省略できますね

Reply:
引数が一つの時であっても（）を省略すると`npm run lint`で警告が出るのでそのままにしています🙏